fix: add evaluate element click for closeWhatsNewModal

marijanabevanda opened the pull request:
<!--
Thank you for your pull request. Please provide all required information
-->

**Short description of work done**
Fixing 'Node is Detached from Document' Error in Puppeteer/Dappeteer Code: Refactoring closeWhatsNewModal Logic

### PR Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [ ] I have run linter locally
- [ ] I have run unit and integration tests locally
- [ ] Update configuration the newest version (readme and const)
- [ ] Rebased to master branch / merged master

### Changes
<!-- Please describe all changes made to codebase. -->
I encountered an intermittent 'Node is detached from document' error while using the closeWhatsNewModal method. This issue occurred sporadically due to the dynamic nature of the application, where the close button was deleted and recreated.

To address this problem, I implemented a solution similar to what was used in the closePopup method. Specifically, I introduced a new method called evaluateElementClick that contains the necessary logic. This method includes a 1-second delay to handle scenarios where elements are dynamically recreated. Both closePopup and closeWhatsNewModal methods now use evaluateElementClick.

<!-- ### Example -->
<!-- You can add screenshots or videos to show changed behaviour -->
Error solved:
![image](https://github.com/ChainSafe/dappeteer/assets/54986853/e6cceb08-1911-4c27-b55b-34c34d5c881d)


### Issues
<!-- Use github keyword to close issues that are related to this PR -->

<!-- [REQUIRED] -->
Closes #

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ChainSafe/dappeteer?pullRequest=345) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ChainSafe/dappeteer?pullRequest=345) before we can accept your contribution.<br/><hr/>**mbevanda** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ChainSafe/dappeteer?pullRequest=345) it.</sub>

marijanabevanda commented:
> **Short description of work done**
fix: Fixing 'Node is Detached from Document' Error in Puppeteer/Dappeteer Code: Refactoring closeWhatsNewModal Logic
> 
> ### PR Checklist
> * [ ]  I have run linter locally
> * [ ]  I have run unit and integration tests locally
> * [ ]  Update configuration the newest version (readme and const)
> * [ ]  Rebased to master branch / merged master
> 
> ### Changes
> I encountered an intermittent 'Node is detached from document' error while using the closeWhatsNewModal method. This issue occurred sporadically due to the dynamic nature of the application, where the close button was deleted and recreated.
> 
> To address this problem, I implemented a solution similar to what was used in the closePopup method. Specifically, I introduced a new method called evaluateElementClick that contains the necessary logic. This method includes a 1-second delay to handle scenarios where elements are dynamically recreated. Both closePopup and closeWhatsNewModal methods now use evaluateElementClick.
> 
> Error solved: ![image](https://user-images.githubusercontent.com/54986853/268900233-e6cceb08-1911-4c27-b55b-34c34d5c881d.png)
> 
> ### Issues
> Closes #
